setup.py

```diff
@@ -29,44 +39,27 @@
 
 def get_parser_ext():
   """Get the parser extension."""
+  extra_compile_args = []
   # We need some platform-dependent compile args for the C extensions.
   if sys.platform == 'win32':
     # windows does not have unistd.h; lex/yacc needs this define.
-    extra_compile_args = ['-DYY_NO_UNISTD_H']
-    extra_link_args = []
-  elif sys.platform == 'darwin':
-    # clang on darwin requires some extra flags, which gcc does not support
-    extra_compile_args = ['-std=c++11', '-stdlib=libc++']
-    extra_link_args = ['-stdlib=libc++']
-  else:
-    extra_compile_args = ['-std=c++11']
-    extra_link_args = []
-  return Extension(
+    extra_compile_args.append('-DYY_NO_UNISTD_H')
+
+  return Pybind11Extension(
```

Comment:
The parser_ext doesn't use pybind11. I assume this will still work fine, but should be remain a normal Extension anyway?

Reply:
This is just a lightweight subclass of Extension that adds pybind11's flags, the cxx_std option, and include dir. It was carefully designed to work anywhere a normal Extension works and does not require a special `build_ext`.